reverse-proxy.md

```diff
@@ -393,19 +393,19 @@ server {
 
 ```
 
-⚠️ **Please note:** Look into [this](#adapting-the-sample-web-server-configurations-below) to adapt the above example configuration.
+⚠️ **Please note:** look in [here](#adapting-the-sample-web-server-configurations-below) to adapt the above example configuration.
```

Comment:
I find it weird, but I'm not a native English speaker so I'm not sure.

Reply:
Also not sure but lets leave it like this for now